### DISCUSSION TITLE
ruTorrent RPC Plugin route

### DISCUSSION THREAD

**Discussion by raulgbcr** [linuxserver/Heimdall#748] [App issues]
Hi, the enhanced app of ruTorrent only allows for the RPC2 path in the host, wich actually enforces at the end when entering the host and port. 

Can we remove the RPC2 path enforcement so we can use the built in ruTorrent rpc plugin too? (/rutorrent/plugins/rpc/rpc.php)

Regards.

**Comment by limpep** [comment]
Having same issue

**Comment by BzowK** [comment]
Same here...

**Comment by mariusberget92** [comment]
Same. Got a total different route than the endpoint being RPC or RPC2

**Comment by Hipoglos** [comment]
Did anybody found the way to fix it?
